### ISSUE TITLE
Metallic Factor is 0 by default

### ISSUE THREAD

**Issue by Safojas** [AsoboStudio/glTF-Blender-IO-MSFS#271]
### Current Behavior

Metallic Factor is 0 by default

### Expected Behavior

Metallic Factor should be 1 for correct display in sim. 
![изображение](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/5460847/f08ffc5b-0337-4e51-8e4a-ef5fb782f551)

### Steps To Reproduce

Create clean material make it msfs material. Check Metallic factor value.

### Environment

```markdown
- OS: windows
- Blender:3.3.9
- glTF-Blender-IO-MSFS: 1.3.1
```

**Comment by Safojas** [comment]
@ronh991 If you have any metallic map and multiply it by 0 you got 0 (black map) with isn’t correct.

**Comment by y-khodja** [comment]
Hello,

It has been fixed in the latest release 1.3.2.

Thank your for your report
